### PR TITLE
Implement ButtonWithAuthz component

### PR DESCRIPTION
**Pull request by Pe5h4**
This PR implements component ButtonWithAuthz. The usage is provided within incode description. The button extends usage of reactstrap Button by disable feature based on resource

### CODE REVIEW

**Comment by Pe5h4** [comment]
Button with authz implemented in LogMan.io WebUI

authzbutton

File: src/containers/ButtonWithAuthz.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import { Button } from 'reactstrap';
```
Comment (ateska) [inline src/containers/ButtonWithAuthz.js:2]: Should be in `asab-webui/src/modules/auth/`

File: src/containers/ButtonWithAuthz.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+*/
+
+export function ButtonWithAuthz(props) {
+	let disabled = props.resources.indexOf(props.resource) == -1;
+	let title = props.title;
+
+	// Check on title eventually passed in the props
+	if (disabled) {
+		title = props.titleUnauthorized;
```
Comment (ateska) [inline src/containers/ButtonWithAuthz.js:54]: I suggest to include a "generic unauthorised message" here ...

File: src/containers/ButtonWithAuthz.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+			color={props.color}
+			outline={props.outline}
+			innerRef={props.innerRef}
+			onClick={props.onClick}
+			size={props.size}
+			className={props.className}
+			cssModule={props.cssModule}
+			onSubmit={props.onSubmit}
+			title={title}
+			disabled={disabled}
```
Comment (ateska) [inline src/containers/ButtonWithAuthz.js:76]: how about `..props` instead of that list?

**Review by ateska — CHANGES_REQUESTED** [review]

**Comment by Pe5h4** [comment]
Requested changes made - component `ButtonWithAuthz` has been moved to `auth` module. Generic `unauthorized message` has been set. Passing props to button has been modified.

File: src/modules/auth/ButtonWithAuthz.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+*/
+
+export function ButtonWithAuthz(props) {
+	const { t, i18n } = useTranslation();
+	let disabled = props.resources.indexOf(props.resource) == -1;
+
+	// Check on title eventually passed in the props
+	if (disabled) {
+		props["title"] = t("You do not have rights");
```
Comment (ateska) [inline src/modules/auth/ButtonWithAuthz.js:59]: "You do not have access rights to perform this action."

**Review by ateska — CHANGES_REQUESTED** [review]
Change in text, then mergeable.

File: src/modules/auth/ButtonWithAuthz.js (hunk 4 above)
Comment (Pe5h4) [reply src/modules/auth/ButtonWithAuthz.js]: Changed

**Review by ateska — APPROVED** [review]

**Pull request merged**